index.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
+  <html lang="en">
+    <head>
+      <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <link rel="stylesheet" href="./css/style.css">
+      <title>JS Documentation</title>
+    </head>
+
+    <body>
+      <nav id="navbar">
+        <header>JS Documentation</header>
+        <ol>
+            <li a class="nav-link" a href="#Introduction">Introduction</li>
+            <li a class="nav-link" a href="#What_you_should already_know">What you should already know</li>
+            <li a class="nav-link" a href="#JavaScript_and_Java">JavaScript and Java</li>
+            <li a class="nav-link" a href="#Hello_world">Hello world</li>
+            <li a class="nav-link" a href="#Variables">Variables</li>
+            <li a class="nav-link" a href="#Declaring variables">Variables</li>
+
+        </ol>
+    </nav>
+      <main id="main-doc">
+        <section class="main-section" id="Introduction">
+          <header>Introduction</header>
+          <p>
+            JavaScript is a cross-platform, object-oriented scripting language. It is a small and lightweight language.
+            Inside a host environment (for example, a web browser), JavaScript can be connected to the objects of its
+            environment to provide programmatic control over them.
+          </p>
+          <p>
+            JavaScript contains a standard library of objects, such as Array, Date, and Math, and a core set of language
+            elements such as operators, control structures, and statements. Core JavaScript can be extended for a variety of
+            purposes by supplementing it with additional objects; for example:
+          </p>
+          <ul>
+            <li>
+              Client-side JavaScript extends the core language by supplying objects to control a browser and its Document
+              Object Model (DOM). For example, client-side extensions allow an application to place elements on an HTML form
+              and respond to user events such as mouse clicks, form input, and page navigation.
+            </li>
+            <li>
+              Server-side JavaScript extends the core language by supplying objects relevant to running JavaScript on a
+              server. For example, server-side extensions allow an application to communicate with a database, provide
+              continuity of information from one invocation to another of the application, or perform file manipulations on
+              a server.
+            </li>
+          </ul>
+        </section>
+
+        <section class="main-section" id="What_you_should already_know">
+
+          <header>What you should already know</header>
```

Comment:
It may be better to use an id for all the header, as well as the paragraphs in each section, in this way, you could use that in CSS to add the styles, please consider the use of id and class parameters in your tags!

Reply:
- [ ] Nice job you did here 👏 . However in the part where you said "Please consider the use of id and class parameters in your tags!", kindly add a "why" afterward to inform the potential student of the necessity of using an `id` and `class` parameter. Stating a "why" when giving reviews helps the code owner to make better decisions next time when faced with the same scenario which then makes he/she a better developer, and you a  better reviewer 👍

Reply:
@Shedrack-Sunday Can you please give me an example of how to add the "why" Statement? I made an addition to the comment but don't know if it is right or if it needs anything else

Reply:
- [ ] Good job in attempting to add reasoning behind your suggestion. ❤️  However, the correct way to explain the **reason** of the review is to provide what is the advantage of your suggestion and how it could improve the quality of the code. 😎 

For example, in this case you might say something along this way: 
**_"The "id" attribute is primarily used for unique identification and precise targeting, while the "class" attribute is used for grouping, reusability, and applying styles or behavior to multiple elements. Both attributes play crucial roles in creating well-structured and maintainable HTML and CSS code."_**

Reply:
Now I have updated the comment, let me know if there is anything else I can add or if I can improve my review in any other way!